blocks/search/graphql-api.js

```diff
@@ -29,7 +29,16 @@ async function fetchGraphQLData(graphqlQuery, endpoint) {
 }
 
 export async function fetchSearchResults({ query, offset, make, model, searchType, category }) {
-  const { SEARCH_URL_DEV, CROSS_REFERENCE_QUERY_NAME, PART_NUMBER_QUERY_NAME, MAX_PRODUCTS_PER_QUERY } = SEARCH_CONFIG;
+  const {
+    SEARCH_URL_DEV = false,
```

Comment:
No need for the false default assignment.
The next check you do will work without that.
same for the other cases

Reply:
done